clients/vscode/src/chat/WebviewHelper.ts

```diff
@@ -424,6 +434,61 @@ export class WebviewHelper {
       });
     };
 
+    const findSymbolInfo = async (filepaths: string[], keyword: string): Promise<SymbolInfo | undefined> => {
```

Comment:
instead of finding symbol across all filepaths, would it be sufficient to just use `executeDefinitionProvider` under context of current **active** tab?

Reply:
I prefer to keep using all file paths so that the symbol navigation event can still work even if all active tabs are closed. This ensures the same behavior as similar tools.

Reply:
it will only passing active selection context filepath now.  `ee/tabby-ui/components/message-markdown/index.tsx`